### PR TITLE
Add LVMLogicalVolume support in MetalK8s UI

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: UI, volumes

**Context**: 

**Summary**:
- LVMLogicalVolume type
- VG Name to replace the Backend Disk
- Add integration test for LVMLogicalVolume

**Acceptance criteria**: 

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: ui/cypress/integration/volume/volumetabs.spec.js [hunk 1]
```diff
@@ -44,6 +44,12 @@ describe('Volume page overview tab', () => {
       );
     });
   });
+
+  it('displays the LVMLogicalVolume type and VG Name', () => {
```
Comment (gdemonet) [inline ui/cypress/integration/volume/volumetabs.spec.js]: We're missing "negative testing", basically:
- not showing "VG Name" on sparse volumes
- not showing "Backend Disk" on LV volumes

Can be handled as debt.

File: ui/cypress/fixtures/kubernetes/volumes.json [hunk 2]
```diff
@@ -1144,6 +1144,118 @@
...
+                            "f:conditions": {},
+                            "f:deviceName": {}
+                        }
+                    },
+                    "manager": "storage-operator",
+                    "operation": "Update",
+                    "time": "2021-04-21T07:52:07Z"
+                }
+            ],
+            "name": "mongodb-volume",
```
Comment (gdemonet) [inline ui/cypress/fixtures/kubernetes/volumes.json:1224]: Would make sense to add this object in the same commit that updates the test for using it.

**Review by gdemonet — DISMISSED** [review]
Only minor comments, just so we don't repeat these mistakes in the future :wink: Feel free to not tackle them right now

File: ui/cypress/integration/volume/volumetabs.spec.js (hunk 1 above)
Comment (ChengYanJin) [reply ui/cypress/integration/volume/volumetabs.spec.js]: Added the two scenarios ;)

File: ui/cypress/fixtures/kubernetes/volumes.json [hunk 3]
```diff
@@ -1144,6 +1144,118 @@
...
           "deviceName": "loop3"
         }
+      },
+      {
+        "apiVersion": "storage.metalk8s.scality.com/v1alpha1",
+        "kind": "Volume",
+        "metadata": {
+            "annotations": {
+                "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"storage.metalk8s.scality.com/v1alpha1\",\"kind\":\"Volume\",\"metadata\":{\"annotations\":{},\"name\":\"mongodb-volume\"},\"spec\":{\"lvmLogicalVolume\":{\"size\":\"50Gi\",\"vgName\":\"vgartesca\"},\"nodeName\":\"installer.novalocal\",\"storageClassName\":\"ssd-raid1-ext4\",\"template\":{\"metadata\":{\"labels\":{\"app.kubernetes.io/name\":\"mongodb\",\"app.kubernetes.io/part-of\":\"zenko\"}}}}}\n"
+            },
```
Comment (TeddyAndrieux) [inline ui/cypress/fixtures/kubernetes/volumes.json]: I think you can remove this annotation you do not have it in other volumes

**Review by gdemonet — APPROVED** [review]

**Pull request merged**